src/Session.hs

```diff
@@ -70,9 +71,8 @@ hspecCommand = "Test.Hspec.Runner.hspecResult spec"
 hasSpec :: Session -> IO Bool
 hasSpec Session{..} = do
   xs <- normalizeTypeSignatures <$> eval sessionInterpreter (":type " ++ hspecCommand)
-  case lines xs of
-    [ys] -> return $ (hspecCommand ++ " :: IO ") `isPrefixOf` ys && "Summary" `isSuffixOf` ys
-    _ -> return False
+  let match line = (hspecCommand ++ " :: IO ") `isPrefixOf` line && "Summary" `isSuffixOf` line
+  return . any match . lines $ xs
```

Comment:
Again, it would be nice to test this change.  Probably it makes sense to split `hasSpec` into a pure and an `IO` part and focus on testing the pure part.